routers/user/auth.go

```diff
@@ -97,7 +97,11 @@ func checkAutoLogin(ctx *context.Context) bool {
 			ctx.SetCookie("redirect_to", "", -1, setting.AppSubURL)
 			ctx.Redirect(redirectTo)
 		} else {
-			ctx.Redirect(setting.AppSubURL + "/")
+			if setting.LandingPageURL != setting.LandingPageHome {
```

Comment:
I think there is no need for this as this would do the same: `ctx.Redirect(setting.AppSubURL + string(setting.LandingPageURL))`

Reply:
Agreed.